standalone/src/main/java/io/pravega/local/InProcPravegaCluster.java

```diff
@@ -332,11 +332,14 @@ private ControllerServiceMain startLocalController(int controllerId) {
                 .tokenSigningKey("secret")
                 .build();
 
-        RESTServerConfig restServerConfig = RESTServerConfigImpl.builder()
-                .host("0.0.0.0")
-                .port(this.restServerPort)
-                .build();
-
+        RESTServerConfig restServerConfig = null;
+        if (this.startRestServer) {
```

Comment:
Please expose the setting for `LocalPravegaController` and set it using the config file. This will help users to tune `standalone` to use this setting.

Reply:
Thanks for the review @arvindkandhare. I've made the changes. Can you please review it again?